### DISCUSSION TITLE
About missing code

### DISCUSSION THREAD

**Discussion by feifei325** [nielsbasjes/yauaa#583] [Q&A] [answered]
# About missing code

Hello, when I download the code from the master clone, I found that some functions are missing in the parser directory, which makes it impossible to compile and run.

* screenshot of idea

![image](https://user-images.githubusercontent.com/46489071/170018497-bb2b7577-878c-4732-94fe-e0fe3ecf334b.png)

The second confirmation on GitHub does lose some functions[github](https://github.com/nielsbasjes/yauaa/tree/main/analyzer/src/main/java/nl/basjes/parse/useragent/parse). How can I modify it to pack normally? 

Thank you!

**Comment by nielsbasjes** [comment, marked as answer]
A lot of code is generated as part of the build. IntelliJ does not run these steps automatically. Easiest way to solve this is to run a mvn compile which should generate what you need.

**Reply by feifei325** [reply]
Thank you for your help. I have solved this problem.
